js/modules/k6/grpc/client.go

```diff
@@ -108,6 +108,14 @@ func (c *Client) Connect(addr string, params map[string]interface{}) (bool, erro
 	ctx, cancel := context.WithTimeout(c.vu.Context(), p.Timeout)
 	defer cancel()
 
+	if p.MaxReceiveSize != 0 {
+		opts = append(opts, grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(int(p.MaxReceiveSize))))
+	}
+
+	if p.MaxSendSize != 0 {
```

Comment:
```suggestion
	if p.MaxSendSize > 0 {
```
Why do we accept negative numbers here? Are they good for some reason?

Reply:
Not that I know of, but didn't think too much about it since the gRPC client allow those negative values.
I can add a check for a positive int when parsing the params if you think it's the best.